CheckMate.egg-info/PKG-INFO

```diff
@@ -0,0 +1,41 @@
+Metadata-Version: 2.1
```

Comment:
I'm not sure if any of the `egg.info` files should be committed since they're generated as part of the package installation process, so it's different for each installation